src/App.vue

```diff
@@ -302,19 +293,19 @@ export default class App extends Vue {
   @Action setWindowWidth!: ActionBindingIF
 
   // Local properties
-  protected accountAuthorizationDialog = false
```

Comment:
We won't be using access modifiers moving on Sev?

Reply:
That depends on Typescript and the VS Code Vue extensions that we use (like volar and eslint).

At the moment, TS ignores access modifiers, so privates aren't really privates. Also, currently, volar and eslint generate errors when a Vue file's template uses a method/property that isn't public -- imho a protected should be fine since the template is sort-of part of the class. So, right now, methods/properties used in the template need to be public to avoid apparent errors.

Maybe in the future we can tighten up our access modifiers. We don't use classes as in C++ but good OO practises are generally a good idea.

Reply:
Noted! I'll keep that in mind. Thanks Sev 👍